src/SFML/Window/WindowBase.cpp

```diff
@@ -207,18 +211,67 @@ void WindowBase::setSize(const Vector2u& size)
 {
     if (m_impl)
     {
-        m_impl->setSize(size);
+        // Constrain requested size within minimum and maximum bounds
+        const auto minimumSize = m_impl->getMinimumSize().value_or(sf::Vector2u());
+        const auto maximumSize = m_impl->getMaximumSize().value_or(
+            sf::Vector2u(std::numeric_limits<unsigned int>::max(), std::numeric_limits<unsigned int>::max()));
```

Comment:
Are `{}` not enough to init a `sf::Vector2u`?

Reply:
`sf::` namespace is not needed here

Reply:
> Are {} not enough to init a sf::Vector2u?

For whatever reason, no.